### PR TITLE
initial k8s 1.22 deprecation changes

### PR DESCRIPTION
**Pull request by tobybellwood**

### CODE REVIEW

File: charts/aergia/values.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ image:
   repository: amazeeio/aergia
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: k8s_122
```
Comment (tobybellwood) [inline charts/aergia/values.yaml]: do we have an updated image for 1.22 - the appVersion says v0.1.2? - we should remove this tag if so?

File: charts/dbaas-operator/values.yaml [hunk 2]
```diff
@@ -84,7 +84,7 @@ image:
   repository: amazeeio/dbaas-operator
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: k8s_122
```
Comment (tobybellwood) [inline charts/dbaas-operator/values.yaml]: do we have an updated image for 1.22 - the appVersion says v0.3.0? - we should remove this tag if so?

File: charts/dioscuri/values.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ image:
   repository: amazeeio/dioscuri
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: k8s_122
```
Comment (tobybellwood) [inline charts/dioscuri/values.yaml]: do we have an updated image for 1.22 - the appVersion says v0.2.0? - we should remove this tag if so?

File: charts/aergia/values.yaml (hunk 1 above)
Comment (shreddedbacon) [reply charts/aergia/values.yaml]: Probably remove it

**Pull request merged**